src/ripple/basics/impl/StringUtilities.cpp

```diff
@@ -31,14 +32,11 @@
 
 namespace ripple {
 
-// NIKB NOTE: This function is only used by strUnHex (std::string const& strSrc)
-// which results in a pointless copy from std::string into std::vector. Should
-// we just scrap this function altogether?
-int strUnHex (std::string& strDst, std::string const& strSrc)
+std::pair<Blob, bool> strUnHex (std::string const& strSrc)
```

Comment:
The return from `unhex` can be ignored - you pass the iterator in, and it's only pointing to one past the last element written.

Point taken about the exceptions.